README.md

```diff
@@ -302,6 +303,146 @@ Let's say we have an instance in the database:
         });
       ```
 
+# Force Id
+
+In loopback `forceId` means user can specify the value of the primary key when creating a model instance, instead of using an auto-generated one. Learn more about LoopBack's [forceId](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).
+
+Every document stored in Cloudant has an unique id, stored as `_id`, and the database has built-in index for it. Retrieving data by its `_id` gives a better performance than querying by other field. Therefore, the connector always assign the `_id` field as the loopback model's primary key.
+
+If you have two models that allow customized `_id` as primary key set with ```"forceId": false``` in the model definition, it could result in a document update conflict.
+
+For example:
+```
+{_id: 'myid', loopback__model__name: 'Foo'}
+{_id: 'myid', loopback__model__name: 'Bar'}
+```
+
+We recommend user to be careful when creating customized model id instead of using the auto-generated one because data belonging to different models can interfere with each other.
```

Comment:
wait...I am not sure why git shows I added on this line, it should be after line 308:
https://github.com/strongloop/loopback-connector-cloudant/pull/174/files#diff-04c6e90faac2675aa89e2176d2eec7d8R308, the sequence would be like this:

In loopback `forceId` means user can specify the value of the primary key when creating a model instance, instead of using an auto-generated one. Learn more about LoopBack's [forceId](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).

We recommend user to be careful when creating customized model id instead of using the auto-generated one because data belonging to different models can interfere with each other.